Dockerfile

```diff
@@ -1,57 +1,31 @@
 #Harvester + Store run in the same container as the store is merely a thin layer on top of the former
 FROM proycon/codemeta-harvester
 
-LABEL org.opencontainers.image.authors="Maarten van Gompel <proycon@anaproy.nl>"
-LABEL description="CLARIAH Tool Store & Harvester"
-
-ARG BASEURL="https://tools.clariah.nl/"
-ENV BASEURL=$BASEURL
-ENV CODEMETA_BASEURI=$BASEURL
-ENV CODEMETA_TITLE="Tools"
-ENV CODEMETA_CSS="${BASEURL}static/extra.css"
-ENV CODEMETA_INPUTLOGDIR="/tool-store-data/"
-
-ARG CRON_HARVEST_INTERVAL="3 * * * *"
-ENV CRON_HARVEST_INTERVAL=$CRON_HARVEST_INTERVAL
-
-#You will want to pass this at run time:
-#ENV GITHUB_TOKEN
-
 ENV GIT_TERMINAL_PROMPT=0
-
-ENV SOURCE_REGISTRY_REPO="https://github.com/CLARIAH/tool-discovery.git"
-#Path within the above repository where the registry is located
-ENV SOURCE_REGISTRY_ROOT="source-registry"
-
 RUN mkdir -p /var/www/static && cp /usr/lib/python3.*/site-packages/codemeta/resources/* /var/www/static/
 
-#copy additional static resources
-COPY static/* /var/www/static/
-
 #Install webserver and build dependencies
-RUN apk add nginx ca-certificates runit cronie rsync py3-dotenv gcc libc-dev make python3-dev
+#Install codemeta-server, this also pulls in rdflib-endpoint and uvicorn (for which we need the build dependencies)
+#remove build dependencies
+RUN apk add bash nginx ca-certificates runit cronie rsync py3-dotenv gcc libc-dev make python3-dev ; \
+pip install git+https://github.com/proycon/codemeta-server flask waitress ; \
```

Comment:
pip install codemeta-server is better!
I don't know if different version will have an impact but I'm afraid that we're not using fixed versions in general. 
So updates can give use problems. Using a requirements.txt with fixed versions can help.
Just for example, I tried to rebuild codemeta-harvester image and starting from this I tried to build this tool-discovery image:  the pip install step on  gave me "can't find rust compiler" error for "'maturin>=0.12,<0.13'".
Even a pip install --upgrade pip before it didn't fix the situation

Reply:
> I don't know if different version will have an impact but I'm afraid that we're not using fixed versions in general.

I see your point yes. Other versions might have an unexpected impact when the next Alpine Linux release comes out yeah (since the base image is `alpine:latest`), but that also adds some pressure (on me) to properly maintain the container image and not let it get too out of date, which is good I think (don't want to miss out on security updates) :)

> pip install step on gave me "can't find rust compiler" error for "'maturin>=0.12,<0.13'".

I wonder why you got that, we're not using anything rust-based I think.

Reply:
I tried to reproduce the maturin error you got, but I can't seem to, everything works fine. Do you have a longer log to show perhaps?

Reply:
It seems your `alpine:latest` image is out of date (still on 3.15, 3.16 was released recently). Can you try upgrade (`docker pull alpine:latest`) and see if that solves the problem?

Reply:
> It seems your `alpine:latest` image is out of date (still on 3.15, 3.16 was released recently). Can you try upgrade (`docker pull alpine:latest`) and see if that solves the problem?

Working now :)